README.md

```diff
@@ -221,6 +221,57 @@ public class BookFilter : QueryableFilter<Book>
 }
 ```
 
+### Nested view-models
+
+`uni-data-table` is capable of handling view-models that are nested in the row view-model.
+
+```json
+{
+  "FirstName": "",
+  "Email": {
+    "Address$": ""
+  },
+}
+```
+
+
+To do so you have to register new column name for the nested property:
+
+```c#
+this.DataTable = new DataTablePage();
+dataTablePage.DataTable = new DataTableBuilder<DataTableRowViewModel>()
+    .WithDataSource(DbLinq.Objects<DataTableRowDataModel>())
+    .WithColumns(columns =>
+        columns
+            .AddColumn(b => b.FirstName, column => column.DisplayName("Name"))
+            .AddColumn(b => b.Email.Address, column => column.DisplayName("Email")))
+    .Build();
+```
+
+and then add custom column display in the client side implementation:
```

Comment:
I have a feeling that the client side implementation example should be not in this README, but in https://github.com/Starcounter/uniform/tree/master/components/uni-data-table (with a cross-link). WDYT?